client/gutenberg/extensions/map-block/settings.js

```diff
@@ -0,0 +1,94 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+
+export const settings = {
+	name: 'jetpack/maps',
+	title: __( 'Map', 'jetpack' ),
+	icon: (
+		<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+			<path d="M20.5 3l-.16.03L15 5.1 9 3 3.36 4.9c-.21.07-.36.25-.36.48V20.5c0 .28.22.5.5.5l.16-.03L9 18.9l6 2.1 5.64-1.9c.21-.07.36-.25.36-.48V3.5c0-.28-.22-.5-.5-.5zM15 19l-6-2.11V5l6 2.11V19z" />
+			<path d="M0 0h24v24H0z" fill="none" />
+		</svg>
+	),
+	category: 'common',
```

Comment:
Beforr shipping with Jetpack preset, let's make sure to have this as 'jetpack' so that it'll appear under jetpack category. We can later look into making this dynamic for Atavist using the SDK.

Reply:
This was actually just done in https://github.com/Automattic/wp-calypso/pull/27836/commits/3306a87b33357df4ab1b8216829b2cfdc0046252